src/brogue/Globals.c

```diff
@@ -91,6 +91,7 @@ const color darkBlue =              {0,     0,      50,     0,      0,
 const color darkTurquoise =         {0,     40,     65,     0,      0,          0,          0,      false};
 const color lightBlue =             {40,    40,     100,    0,      0,          0,          0,      false};
 const color pink =                  {100,   60,     66,     0,      0,          0,          0,      false};
+const color darkPink =              {45,    10,     15,     0,      0,          0,          0,      false};
```

Comment:
Did you define this colour manually? It looks slightly off when dimmed - I think it should be the result of applyColorAverage with black, which seems to be just all values halved

Reply:
Yeah I defined the color manually but colors are not my thing so I'd welcome any help here getting it right. There was a user suggestion that the color be the same as the negation bolt, which is pink, so that was the starting point.

I do agree that the negation definition you came up with is quite elegant. I rebased so should be no conflicts now.

Reply:
The colour in general is fine, I'm talking explicitly about the darkPink for the dimmed text, it needs to be (50, 30, 33)

Reply:
Oh I see now. Ok I updated the values.